src/main/java/cloud/fogbow/rcs/constants/ConfigurationPropertyKeys.java

```diff
@@ -3,7 +3,13 @@
 public class ConfigurationPropertyKeys {
 
     public static final String BUILD_NUMBER_KEY = "build_number";
-    public static final String LOCAL_MEMBER_ID_KEY = "provider_id";
+    public static final String LOCAL_MEMBER_ID_KEY = "provider_id"; // FIXME change this constant to PROVIDER_ID_KEY
```

Comment:
does this fixme still hold?

Reply:
Yes, I was thinking of performing this repair on a refactoring branch or while implementing unit tests.

Reply:
i like the idea of a refactoring pr. the tests specific code should sit in its own pr.